airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -14,10 +14,14 @@ data:
   registries:
     cloud:
       enabled: true
-      dockerImageTag: 1.4.1
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: "Version 2.0.0 changes the schema for most of the Hubspot streams. This is done to unnest deeply nested data so it is available as a column in the new Destinations V2 regarding normalization is no more supported. A schema refresh of all the streams is required as well as resetting the data to use the new version."
+        upgradeDeadline: "2023-10-01"
```

Comment:
this is not a breaking change anymore since we decided to not remove the columns, but duplicate the original data with unnested columns, so I removed this section from metadata.yaml